warehouse/templates/manage/team/roles.html

```diff
@@ -105,6 +109,9 @@ <h3>{% trans %}Add member{% endtrans %}</h3>
       <div id="user-errors">
         {{ field_errors(form.username) }}
       </div>
+      <p class="form-group__help-text">
+        {% trans manage_organization_roles_link=request.route_path('manage.organization.roles', organization_name=team.organization.normalized_name) %}Add an organization owner, manager, or member to the team. <a href="{{ manage_organization_roles_link }}">Manage organization members</a> to view all users in the organization or to invite a user to the organization.{% endtrans %}
```

Comment:
extract link/url: https://warehouse.pypa.io/translations.html#passing-non-translatable-values-to-translated-strings

Reply:
Fixed with commit [d98ae9c](https://github.com/pypi/warehouse/pull/12250/commits/d98ae9c07bddbcf04b27739986c934b23ba652ce).